plugins/inputs/graylog/README.md

```diff
@@ -0,0 +1,42 @@
+# GrayLog plugin
+
+The Graylog plugin can collect data from remote Graylog service URLs. 
+
+
+Sample configration
+```
+[[inputs.graylog]]
+  ## NOTE This plugin only reads numerical measurements, strings and booleans
+  ## will be ignored.
+
+  ## a name for the service being polled
+  name = "graylog_jvm"
+
+  ## URL of each server in the service's cluster
+  servers = [
+    "http://10.224.162.16:12900/system/metrics/multiple"
+  ]
+
+
+  ## List of tag names to extract from top-level of JSON server response
+  tag_keys = [
+  ]
+
+  metrics = [
+    "jvm.cl.loaded",
+    "jvm.memory.pools.Metaspace.committed"
+  ]
+  ## User name and password  
+  [inputs.graylog.auth_key]
```

Comment:
don't make inputs.graylog.auth_key a map, just use two arguments `username` and `password` directly in the plugin